<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>482</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAbort">
    <property name="geometry">
     <rect>
      <x>664</x>
      <y>482</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Anuluj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="surnameLineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="peselLineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Imie</string>
    </property>
   </widget>
   <widget class="QLabel" name="surnameLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nazwisko</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sexLineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneLineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="cityCodeLineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>320</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="cityLineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="streetLineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="peselLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PESEL</string>
    </property>
   </widget>
   <widget class="QLabel" name="sexLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Płeć</string>
    </property>
   </widget>
   <widget class="QLabel" name="phoneLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Telefon</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="cityCodeLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kod pocztowy</string>
    </property>
   </widget>
   <widget class="QLabel" name="cityLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Miejscowosc</string>
    </property>
   </widget>
   <widget class="QLabel" name="streetLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ulica</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="birthDateLineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="birthDateLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Data urodzenia</string>
    </property>
   </widget>
   <zorder>pushButtonAdd</zorder>
   <zorder>pushButtonAbort</zorder>
   <zorder>nameLineEdit</zorder>
   <zorder>peselLineEdit</zorder>
   <zorder>nameLabel</zorder>
   <zorder>surnameLabel</zorder>
   <zorder>sexLineEdit</zorder>
   <zorder>phoneLineEdit</zorder>
   <zorder>emailLineEdit</zorder>
   <zorder>cityCodeLineEdit</zorder>
   <zorder>cityLineEdit</zorder>
   <zorder>streetLineEdit</zorder>
   <zorder>peselLabel</zorder>
   <zorder>sexLabel</zorder>
   <zorder>phoneLabel</zorder>
   <zorder>emailLabel</zorder>
   <zorder>cityCodeLabel</zorder>
   <zorder>cityLabel</zorder>
   <zorder>streetLabel</zorder>
   <zorder>birthDateLineEdit</zorder>
   <zorder>birthDateLabel</zorder>
   <zorder>surnameLineEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>surnameLineEdit</tabstop>
  <tabstop>sexLineEdit</tabstop>
  <tabstop>birthDateLineEdit</tabstop>
  <tabstop>peselLineEdit</tabstop>
  <tabstop>phoneLineEdit</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>cityCodeLineEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>streetLineEdit</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonAbort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
